pygame.py

```diff
@@ -0,0 +1,67 @@
+# Needed to use any and all python resources.
+import pygame
+import pygame_menu  #pip install pygame-menu
+from os import path
+from win32api import GetSystemMetrics 
+import yahtzee        
+
+# Defines common colors
+background_one = (255, 255, 255)  # white
+background_two = (0, 0, 0)  # black
+FPS = 30 # frames per second
+
+# Set the size of the window, variables can be used for positioning
+# within program.
+window_width = GetSystemMetrics((0))
+window_height = GetSystemMetrics((1))
+
+
+#create function to present text
+font_name = pygame.font.match_font('arial') 
+def text_draw(surface, text, size, x, y):
+    font = pygame.font.Font(font_name, size)
```

Comment:
If you google and download a copy of Arial.ttf, and place if within your directory. You can combine lines 20 and 22 to: 
font = pygame.font.SysFont('Arial.ttf', size), and it will work the same.